manifests/install/all-in-one.yaml

```diff
@@ -0,0 +1,83 @@
+# First part
```

Comment:
@cwdsuzhou @denkensk could you help compare the RBAC/CRD with your production settings? to ensure the correctness and the privileges are not over-exposed. Thanks in advance.

Reply:
Hi, coscheduling works ok in my cluster using the the RBAC(changed in this PR) and CRD

![image](https://user-images.githubusercontent.com/12230714/111416625-423d9480-871f-11eb-8bb4-d15d906320e6.png)
  

Since pod group controller is not necessary now, we did not deploy in our production environment.